components/courses/AssessmentsTimeline.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <div>
+    <v-col cols="12" md="12">
+      <div :style="chartStyle">
+        <v-chart autoresize :option="option" @click="goToAssessment" />
+      </div>
+    </v-col>
+  </div>
+</template>
+<script>
+import datetime from '@/mixins/datetime.js'
+import '@/plugins/echarts'
+export default {
+  name: 'AssessmentsTimeline',
+  mixins: [datetime],
+  props: {
+    assessments: {
+      type: Array,
+      required: true,
+    },
+    code: {
+      type: String,
+      required: true,
+    },
+    schedule: {
+      type: Array,
+      default: () => null,
+    },
+  },
+  computed: {
+    chartStyle() {
+      let style = 'width: 100%; height: '
```

Comment:
Prefer using const over let (more efficient). In this case, you can use template literals to simplify the function.